storybook/withTheme.js

```diff
@@ -18,28 +18,44 @@
     51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 */
 import * as React from 'react';
+import { View } from 'react-native';
 import { Provider, connect } from 'react-redux';
 import { ThemeProvider } from 'styled-components';
 import { getThemeByType } from 'utils/themes';
+import type { Dispatch, RootReducerState } from 'reducers/rootReducer';
 
-import type { RootReducerState } from 'reducers/rootReducer';
-
+import { Container } from 'components/Layout';
+import { DARK_THEME, LIGHT_THEME } from 'constants/appSettingsConstants';
+import Button from 'components/Button';
+import { setAppThemeAction } from 'actions/appSettingsActions';
+import { getEnv } from 'configs/envConfig';
 import configureStore from '../src/configureStore';
 
 const { store } = configureStore();
 
 type Props = {
   themeType: string,
   children: React.Node,
+  setAppTheme: (themeType: string) => void,
 };
 
-const StoryWrapper = ({ themeType, children }: Props) => {
+const StoryWrapper = ({ themeType, children, setAppTheme }: Props) => {
   const theme = getThemeByType(themeType);
+  const { current } = theme;
+  const themeToChangeTo = current === LIGHT_THEME ? DARK_THEME : LIGHT_THEME;
 
   return (
-    <ThemeProvider theme={theme}>
-      {children}
-    </ThemeProvider>
+    <View style={{ height: '100%' }}>
+      <ThemeProvider theme={theme}>
+        <Container style={{ flex: 1 }}>
+          {children}
+        </Container>
+      </ThemeProvider>
+      {!!getEnv().SHOW_THEME_TOGGLE && <Button
```

Comment:
A funny little bug I discovered by accident. Go to storybook, open up any Event Details story. Edit Event Detail related files (or just save them). It should trigger app refresh (Fast Refresh). And add a new theme toggle button :D 
 
<img width="415" alt="Screenshot 2020-10-13 at 12 00 50" src="https://user-images.githubusercontent.com/6280457/95846477-c0020c00-0d4b-11eb-8a10-ade2ec47b158.png">

Reply:
Hm, I caught it once.
Will look into it!
But hey, it increases clicking area significantly :))